registry/config.go

```diff
@@ -67,7 +67,7 @@ var lookupIP = net.LookupIP
 // InstallCliFlags adds command-line options to the top-level flag parser for
 // the current process.
 func (options *ServiceOptions) InstallCliFlags(flags *pflag.FlagSet) {
-	mirrors := opts.NewNamedListOptsRef("registry-mirrors", &options.Mirrors, ValidateMirror)
```

Comment:
Oh, interesting; it marks URLs containing a username/password (`http://user:pass@example.com/`) as "valid", but the original version strips them from the "normalised" URL